devices/niko.js

```diff
@@ -232,13 +270,21 @@ module.exports = [
             await reporting.onOff(ep2);
             await ep1.read('manuSpecificNiko1', ['switchOperationMode']);
             await ep2.read('manuSpecificNiko1', ['switchOperationMode']);
+            await ep1.read('manuSpecificNiko1', ['outletLedState']);
```

Comment:
You can probably read switchOperationMode, outletLedState, and outletLedColor in one read.
While messing around with initial support for these I managed to read 5+ attributes at a time. So something like:

```
await ep1.read('manuSpecificNiko1', ['switchOperationMode', 'outletLedState', 'outletLedColor']);
await ep2.read('manuSpecificNiko1', ['switchOperationMode', 'outletLedState', 'outletLedColor']);
```

Reply:
I am getting timeouts reading outletLedState and outletLedColor on the single switch one, but I had one of the very first units so it might have old firmware, it certainly has a ton of other bugs where I am able to completely brick the device.

Given I am not getting UNSUPPORTED_ATTRIBUTE I think it probably crashed the firmware on mine again 😞

Reply:
>Curious to what firmware version yours has though, mine is 313-687-00 (20210224)
That would be the same for all of my zigbee switches.

I did have some issues after trying to read some of the standard device properties, where it would just timeout. But once I physically pressed the switch, everything returned to normal.

Reply:
Firmware on these is definitely not super stable. 😔

Reply:
I did a reset on one of my switches and connected it to my Niko hub for a couple of hours, but the firmware was not updated, so I assume it's probably still the latest version.